cocos2d/core/renderer/assembler-2d.js

```diff
@@ -126,6 +126,8 @@ export default class Assembler2D extends Assembler {
             indiceOffset = offsetInfo.indiceOffset,
             vertexId = offsetInfo.vertexOffset;
         for (let i = 0, l = iData.length; i < l; i++) {
+            //Because taobao-minigame does not allow out-of-bounds access to Uint16Array, otherwise it will report an error and cause the game to be unresponsive
+            if (cc.sys.platform === cc.sys.TAOBAO_MINIGAME && indiceOffset >= ibuf.length) break;
```

Comment:
- why do it in for loop, can determine how much data to copy before for loop
- this PR means that on tabao platform, some data will not be copied into ibuf, can it work correctly on tabao platform?